components/account/AccountSalutationInner.vue

```diff
@@ -0,0 +1,48 @@
+<script setup lang="ts">
+import type {Columns} from '~/types/vueForm/Columns';
+
+withDefaults(
+    defineProps<{
+      cols?: Columns;
```

Comment:
```suggestion
      columns?: Columns;
```

in the following files as well